pkg/util/utils.go

```diff
@@ -38,7 +38,7 @@ import (
 
 // Structure maintaining the connection details to a ManagedCluster
 type ManagedClusterConnection struct {
-	KubeClient                  *kubernetes.Clientset
+	KubeClient                  kubernetes.Interface
```

Comment:
Since this is a critical change to code used throughout the operator, I think we need an acceptance test for this.

Reply:
https://build.verrazzano.io/blue/organizations/jenkins/verrazzano/detail/tbeerbower%2FtestVZ-1250/7/pipeline